.prettierignore

```diff
@@ -2,9 +2,9 @@ node_modules
 **/.next/**
 **/_next/**
 **/dist/**
-packages/next/bundles/webpack/packages/*.runtime.js
-packages/next/bundles/webpack/packages/lazy-compilation-*.js
-packages/next/compiled/**
+packages/next/src/bundles/webpack/packages/*.runtime.js
+packages/next/src/bundles/webpack/packages/lazy-compilation-*.js
+packages/next/src/compiled/**
```

Comment:
IMHO `packages/next/compiled/` should not be moved into `src` folder.

Reply:
This is intentional, it's the checked-in source code, not the part that is published.